sympy/core/basic.py

```diff
@@ -123,6 +122,31 @@ def _cmp_name(x: type, y: type) -> int:
     return (i1 > i2) - (i1 < i2)
 
 
+
+@cacheit
+def _get_postprocessors(clsname, arg_type):
+    # Since only Add, Mul, Pow can be clsname, this cache
+    # is not quadratic.
+    postprocessors = set()
+    try:
+        mappings = _get_postprocessors_for_type(arg_type)
+        for mapping in mappings:
+            f = mapping.get(clsname, None)
+            if f is not None:
+                postprocessors.update(f)
+    except TypeError:
```

Comment:
Why is TypeError raised here?

Reply:
I'm not really sure. The original code had a try except block for a TypeError.